Rajitha/word_letter_match.js

```diff
@@ -0,0 +1,48 @@
+/****************************************************************************************
+ * Create a function that takes in a sentence and a character to find. Return an object 
+ * of each word in the sentence, with the count of the specified character as the value.
+ *
+ * findOccurrences("An APPLE a day keeps an Archeologist AWAY...", "A") ➞ {
+  "an" : 1,
+  "apple" : 1,
+  "a" : 1,
+  "day" : 1,
+  "keeps" : 0,
+  "archeologist" : 1,
+  "away..." : 2
+}
+ ****************************************************************************************/
+const matchWord = (str, letter) => {
+  let cnt = 0;
+  let tempWord = "";  
+  let obj = {};
```

Comment:
Are we replacing the obj? If not, use Const. If you don't understand why, let me know. I'll go over it again